src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/rules/dragHandle/item.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {DraggableSyntheticListeners} from '@dnd-kit/core';
+import {Transform} from '@dnd-kit/utilities';
+
+export interface Props {
+  renderItem(args: {
+    value: Props['value'];
+    listeners: DraggableSyntheticListeners;
+    transform: Props['transform'];
+    transition: Props['transition'];
+    style?: React.CSSProperties;
+    forwardRef?: React.Ref<HTMLElement>;
+  }): React.ReactElement | null;
+
+  value: React.ReactNode;
+  transform?: Transform | null;
+  transition?: string;
+  listeners?: DraggableSyntheticListeners;
+  forwardRef?: React.Ref<HTMLLIElement>;
+  style?: React.CSSProperties;
+}
+
+function Item({
+  transition,
+  transform,
+  forwardRef,
+  value,
+
+  listeners,
+  renderItem,
+  style,
+}: Props) {
+  return renderItem({
+    value,
+    listeners,
+    forwardRef,
+    transform,
+    transition,
+    style,
+  });
+}
+
+export default React.memo(Item);
```

Comment:
This component doesn't seem to be adding much. Could the `renderItem()` function be wrapped with memo() more directly?

Reply:
will update ;-)

Reply:
I decided to remove `React.memo `for now, because it is not making much difference